tqdm.py

```diff
@@ -0,0 +1,42 @@
+from time import sleep
+import datetime as dt
+import sys
+
+
+def Tqdm(iterable):
+    iteration_counter = 1
+    total_iterations = len(iterable)
+    start_time = dt.datetime.now()
+    for obj in iterable:
+        # number of iteration
+        sys.stdout.write('\rIteration {} of {}     '.format(iteration_counter, total_iterations))

```

Comment:
`f-strings`
Also, Why not use `print`?
If it's because of line breaks, you can use a different, more standard way - create a big string and print it in the end.

Reply:
Done